joomla/joomla-cms
# [4.3] Remove dependency on factory

### Summary of Changes
Removes dependency on Factory from the publishing plugin. there should be no impact on the functionality of the plugin

### Testing Instructions
Check the publishing plugin continues to inject the javascript into the articles list view when the plugin is enabled. There should be no change in functionality before and after the PR

### Link to documentations
Please select:
